src/Migrations/Utils/Functions.php

```diff
@@ -279,30 +281,53 @@ public static function create_block_columns(array $attrs, array $inner_blocks):
         $html_content = self::generate_html_content("wp-block-columns");
 
         return self::create_new_block(
-            Constants::BLOCK_COLUMNS,
+            Constants::BLOCK_CORE_COLUMNS,
             $attrs,
             $inner_blocks,
             $html_content['html'],
             $html_content['content']
         );
     }
 
+    /**
+     * Create a new columns block.
+     *
+     * @param array $attrs - The attributes of the block.
+     * @return array - The new columns block.
+     */
+    public static function create_block_p4_columns(array $attrs): array
+    {
+        return self::create_new_block(
+            Constants::BLOCK_P4_COLUMNS,
+            $attrs,
+            [],
```

Comment:
I think it could be great if you just set default values from [here](https://github.com/greenpeace/planet4-master-theme/pull/2363/files#diff-cc008632f1accae161e94059eda5822abcb8582a531da1dc18dc042d3a65d37cL187-L191) and then avoid passing this 3 parameteres which is also repeteated in many parts of the code.

Reply:
I see this 🤔

![Screen Shot 2024-11-28 at 07 33 05](https://github.com/user-attachments/assets/a8072762-4f8d-4e11-ba45-3379ac361093)

But it seems you've just pushed that change 👏

Reply:
Yes! I just updated the function to remove the empty arrays/strings passed as parameters.